pages/cloud-providers.html

```diff
@@ -110,39 +95,10 @@
             </p>
 
             <p>
-                <strong><a href="https://services.siick.fr/">services.siick.fr</a></strong>
+                <strong><a href="https://rss.siick.fr/">services.siick.fr</a></strong>
             </p>
         </div>
 
-        <div class="card flow-small">
-            <div class="card__title">
-                ~vern
-            </div>
-
-            <p class="text-secondary">
-                English, free (donations accepted)
-            </p>
-
-            <p>
-                <strong><a href="https://rss.vern.cc/">rss.vern.cc</a></strong>
-            </p>
-        </div>
-
-        <div class="card flow-small">
-            <div class="card__title">
-                Zaclys
-            </div>
-
-            <p class="text-secondary">
-                French, 10 € / year
-            </p>
-
-            <p>
-                <strong><a href="https://www.zaclys.com">zaclys.com</a></strong>
-            </p>
-        </div>
```

Comment:
Zaclys still provides FreshRSS (see "[Flux](https://www.zaclys.com/flux/)", the screenshot is wrong though)

Reply:
Ahh. I removed it based on the changes and the "Flux" mentioned, which I believed to be Flux reader, and not exactly FreshRSS. Is there a instance page you can direct me to? I will add that as well, as I could not locate one myself.